CMakeLists.txt

```diff
@@ -531,13 +531,13 @@ if (NOT BUILD_SHARED_LIBS)
 else (NOT BUILD_SHARED_LIBS)
   target_compile_definitions (glog PRIVATE GOOGLE_GLOG_IS_A_DLL=1)
 
-  if (HAVE___ATTRIBUTE__VISIBILITY_DEFAULT)
+  if (HAVE___ATTRIBUTE__VISIBILITY_DEFAULT AND NOT MSVC)
```

Comment:
I think we are trying to avoid using vendor names. Maybe better to change the order of conditions?
```
if (HAVE___DECLSPEC)
 set (_EXPORT "__declspec(dllexport)")
 set (_IMPORT "__declspec(dllimport)")
elseif (HAVE___ATTRIBUTE__VISIBILITY_DEFAULT)
  ...
endif(HAVE___DECLSPEC)
```

Reply:
Thank you for your suggestion. I changed it.